requirements.txt

```diff
@@ -1,7 +1,7 @@
 slack_bolt
 Flask>=1.1
 gunicorn>=20
-openai==0.28
+openai
 tiktoken
 trafilatura
 json-logger-stdout
```

Comment:
One thing we could improve later is to use something like `poetry` to manage the dependencies https://python-poetry.org/

That will allow us to lock dependencies in the repo as well